public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -18,60 +19,62 @@ interface Props {
   dashboardUID: string;
 }
 
-export const DashboardLinksDashboard = (props: Props) => {
-  const { link, linkInfo } = props;
-  const listRef = useRef<HTMLUListElement>(null);
-  const [dropdownCssClass, setDropdownCssClass] = useState('invisible');
-  const [opened, setOpened] = useState(0);
-  const resolvedLinks = useResolvedLinks(props, opened);
+interface DashboardLinksMenuProps {
+  link: DashboardLink;
+  dashboardUID: string;
+}
+
+function DashboardLinksMenu({ dashboardUID, link }: DashboardLinksMenuProps) {
   const styles = useStyles2(getStyles);
+  const resolvedLinks = useResolvedLinks({ dashboardUID, link });
 
-  useLayoutEffect(() => {
-    setDropdownCssClass(getDropdownLocationCssClass(listRef.current));
-  }, [resolvedLinks]);
+  if (!resolvedLinks || resolveLinks.length === 0) {
+    return null;
+  }
+
+  return (
+    <Menu>
+      <div className={styles.dropdown}>
+        <CustomScrollbar>
+          {resolvedLinks.map((resolvedLink, index) => {
+            return (
+              <Menu.Item
+                url={resolvedLink.url}
+                target={link.targetBlank ? '_blank' : undefined}
+                key={`dashlinks-dropdown-item-${resolvedLink.uid}-${index}`}
+                label={<div className={styles.label}>{resolvedLink.title}</div>}
+                data-testid={selectors.components.DashboardLinks.link}
+                aria-label={`${resolvedLink.title} dashboard`}
+                // rel="noreferrer"
```

Comment:
```suggestion
```
This is handled by the `MenuItem` component based on the `target` prop.

Reply:
lovely!